src/components/TextField/TextField.scss

```diff
@@ -127,6 +127,13 @@ $stacking-order: (
       margin: 0;
     }
   }
+
+  // Hide webkit search clear button
+  &[type='search'] {
+    &::-webkit-search-cancel-button {
```

Comment:
What happens in FF and IE?

Reply:
+1 to this question, is this related to the `x` that appears in chrome for search inputs? If so, I think this makes sense, but maybe should be scoped to cases where the clear button is present.

Reply:
@danrosenthal Thats right, this is to hide the `x` that appears in chrome / firefox search inputs. I've adjusted it so that it is scoped only to inputs with a clear button. 

Say we have a `TextField` component with `type="search"` and `clearButton={false}` - would we want the browser clearButton to appear in that case? That is how I have it set up right now, but I am not sure if that is the way to go.